src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -36,6 +36,22 @@
 
 namespace Microsoft.ML.Transforms
 {
+    /// <summary>
+    /// <p>A transform for scoring ONNX models in the ML.NET framework.</p>
+    /// <format type="text/markdown">
+    /// <![CDATA[
+    /// [!code-csharp[MF](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/OnnxTransform.cs)]
+    /// ]]>
+    /// </format>
+    /// </summary>
+    /// <remarks>
+    /// <p>Supports inferencing of models in 1.2 and 1.3 format, using the
+    /// <a href='https://www.nuget.org/packages/Microsoft.ML.Scoring/'>Microsoft.ML.Scoring</a> library
```

Comment:
This is out of date, right?  We no longer use `Microsoft.ML.Scoring`.

Reply:
good catch. replaced with onnxruntime.

---
In reply to: [236332004](https://github.com/dotnet/machinelearning/pull/1717#discussion_r236332004) [](ancestors = 236332004)